components/molecules/Checkout/PaymentMethodSection.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react'
+import { RadioGroup, Text, View } from 'tamagui'
+
+import RadioGroupItemWithLabel from '~/components/molecules/common/RadioGroupItemWithLabel'
+import useTranslation from '~/hooks/useTranslation'
+import { PaymentMethod } from '~/interfaces/enum/Payment'
+
+
+const PaymentMethodSection = (): {
+  renderPaymentMethods: () => JSX.Element
```

Comment:
The return object (`renderPaymentMethods` and `selectedMethodID`) may be more suitable for a custom hook rather than a component.